src/SixLabors.Fonts/Tables/AdvancedTypographic/GPos/AnchorTable.cs

```diff
@@ -45,7 +45,8 @@ public static AnchorTable Load(BigEndianBinaryReader reader, long offset)
             return anchorFormat switch
             {
                 1 => AnchorFormat1.Load(reader),
-                _ => throw new NotSupportedException($"anchorFormat {anchorFormat} not supported. Should be '1'.")
+                2 => AnchorFormat2.Load(reader),
```

Comment:
As somebody who knows nothing about fonts, making these error messages more descriptive might be helpful too. If I knew that my font format wasn't supported, I'd just pick a different one.

```csharp
 3 => throw new NotSupportedException($"Variable fonts (format 3) are not supported. Read: https://docs.sixlabors.com/articles/fonts/anchorTableFormats.html")
 _ => throw new NotSupportedException($"Fonts using anchor table format {anchorFormat} are not supported. Read: https://docs.sixlabors.com/articles/fonts/anchorTableFormats.html")
```

Reply:
I wouldn't link to docs from exception messages. The maintenance burden of tracking URL changes would be too much.

Reply:
that's fair enough, was just a thought